tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -32,7 +32,7 @@ def launch(self):
                 *self._settings_button_locator))))
 
     def switch_to_contacts_frame(self):
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        self.wait_to_be_displayed()
         self.apps.switch_to_displayed_app()
 
```

Comment:
if I'll add the manifest_url property, it would be the same for both Phone and Contacts have the same manifest_url and origin: manifestURL: "app://communications.gaiamobile.org…", origin: "app://communications.gaiamobile.org"
For that we need to search for another solution and it's using the entry_point but for I have n idea